app/models/concerns/url_validator.rb

```diff
@@ -1,14 +0,0 @@
-module UrlValidator
```

Comment:
Is there any plugin that uses this?

Reply:
Not in Katello/Foreman org. If there is outside, I've just made them to notice it and take simple recourse.
That's why I chose not to remove the module completely

Reply:
But did not commit, so I can see how my answer could have confuse you xD
Now I've included all the files :)

Reply:
It certainly makes a lot more sense now